DEVELOP.md

```diff
@@ -0,0 +1,43 @@
+# Develop
+
+## Building locally
+
+The data for the dashboard website is sourced from:
+
+- The `activities.yml` file
+- The GitHub issues in mozilla/standards-positions (text in OP, labels)
+
+To fetch the GitHub information:
+
+```
+python3 gh-data.py
+```
+
+This will create or overwrite `gh-data.json` and `gh-data-summary.json`.
+
+Then, to combine that data with the information in `activities.yml`:
+
+```
+python3 merge-data.py
+```
+
+This will create `merged-data.json` which is used by `index.html`.
+
+To view the dashboard page locally, you need to start a local web server:
+
+```
+python3 -m http.server 8000
+```
+
+Then load http://localhost:8000/ in a web browser.
+
+## GitHub Actions / Publishing
+
+Publishing happens automatically when pushing to `main` or when changing labels in GitHub.
+
+The latter runs the `labels-change.yml` workflow directly and then the `schedule-labels-change.yml` workflow periodically checks the history of workflow jobs to determine if labels have changed,
+and if so pushes an empty commit to the `trigger-build` branch,
+which causes the `build-and-deploy.yml` workflow to run.
```

Comment:
This seems a little convoluted.  I can see why you want to avoid running a build on every label change, but this does exactly that.  And I can't see why you have a schedule-labels-change and a build-and-deploy as separate things.

I would have thought that you could just use the API to query for recent changes to issues, filtered for label changes.

Reply:
Hmm, indeed a job is triggered on every label change. I can look into your suggested alternative, I didn't think of that.

Checking for label changes and building is separate because building should happen on changes to `main` even if there were no label changes. Maybe that can be done from a single workflow, though (with `github.event_name`).

Reply:
As far as I can tell you can only get label changes for a single issue at a time, so this would be somewhat complex also. I have now simplified by running `build-and-deploy.yml` on a cron job (every 6 hours), and rely on `git commit` and `git push` not doing anything if there are no changes.